### DISCUSSION TITLE
Prebuilds still active, after months of being deleted

### DISCUSSION THREAD

**Discussion by cgtarmenta** [community/community#60937] [Codespaces]
### Select Topic Area

Bug

### Body

Hello.

Months ago, I tested codespaces, and after a couple of days, I removed all codespaces and the prebuild config.

but have been receiving messages alerting me of this:
![image](https://github.com/community/community/assets/10125168/be4bef21-c72d-4499-bf53-188d8740e54b)

Today they reach 100%, so I decided to investigate.

the report:
![image](https://github.com/community/community/assets/10125168/9ebafb64-8e54-4d41-ab52-b8875279c766)

cgtarmenta repository>settings>codespaces
![image](https://github.com/community/community/assets/10125168/9efc3e89-228c-4913-bfed-350f34df5173)

So it seems that even when all the config is removed, and months have passed, the prebuilds are still active there.

Please, somebody, stop this it is annoying.

**Comment by eljog** [comment]
Our engineering team is looking into this. We are sorry to see that you had to face this issue.

**Comment by gdomingu** [comment]
Hi @cgtarmenta, I looked into this and do see a config associated with the `cgtarmenta` repository and do not see any record of it being deleted. Would you be willing to try to clear your cookies or try a different browser to see if you can see it in the [repository settings](https://github.com/cgtarmenta/cgtarmenta/settings/codespaces)?  If you are still unable to see and delete it, I can delete it for you if that is what you want.

**Reply by cgtarmenta** [reply]
This was captured 4 days ago
![image](https://github.com/community/community/assets/10125168/46add30c-4915-4942-95da-351ca4c2e22c)

I didn't clear anything, and now this config appears
![image](https://github.com/community/community/assets/10125168/40946f90-237d-43e4-8daa-1ef6a619d52a)


![image](https://github.com/community/community/assets/10125168/a159937c-c99b-4caa-973f-50be8b1929c4)
![image](https://github.com/community/community/assets/10125168/3fed4ad1-1c92-4943-895d-2c9b97f59837)

How does this show me only prebuilds from 3 and 4 months ago, however, the report indicates almost daily prebuilds this month?
![image](https://github.com/community/community/assets/10125168/ffb092d8-9938-4f20-9eaa-a1d6ecddaaae)

and even if this configuration is mine, which I highly doubt btw
![image](https://github.com/community/community/assets/10125168/72a5d8c3-b7a6-45fe-9909-f3851f5dbd92)

how is this happening?
![image](https://github.com/community/community/assets/10125168/f631a8b3-2aa5-4809-8565-fa28bfcfa6c2)

when I have no codespace running?
![image](https://github.com/community/community/assets/10125168/895101f2-5009-4cf1-9c0a-89795946fd8a)



You guys have serious bugs there

please, complete and erase any kind of configuration that can remain there, related to codespaces

**Reply by gdomingu** [reply]
@cgtarmenta You should be able to delete the prebuild configuration from the dropdown menu on the [repository settings](https://github.com/cgtarmenta/cgtarmenta/settings/codespaces)
<img width="800" alt="Screenshot of delete prebuild configuration button" src="https://github.com/community/community/assets/4443635/ca1f6105-3cfe-4708-9528-b44bdeeb4788">

I believe the report is showing `Prebuild storage` billing, not the Github Actions minutes from a prebuild workflow running. Please see 
https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#storage-costs-for-prebuilds for more details. 

I do understand how frustrating this could be though, so I will brainstorm with the team on how we can prevent other users from running into a similar issue in the future.

**Comment by cgtarmenta** [comment]
@gdomingu Thanks, I know where the button is, just I don't trust it anymore, that's why I've asked you to delete the thing.

I know that reports show the prebuild storage billing, and that's the entire point of all of this, my prebuild storage usage is been progressively increasing over the past 3 months, until it reached 100% a few days back. I don't understand how that happens without new prebuilds being triggered.

Also, I know this doesn't happen only to me, actually, I've been pointed to open a discussion here by another user with the same symptoms.

I just pressed the delete button. Thanks again.

Cheers.
